### PR TITLE
add membership request issue template

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Add an issue template, easy for developers to apply for kurator membership
**Which issue(s) this PR fixes**:
Fixes # #498

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/membership-request.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+## List of contributions to the Kurator project
+- PRs reviewed / authored:
+
+- Issues authored:
+
+- Issues responded to:
+
+
+<!-- Please don't edit the following lines -->
+[community membership guidelines]: https://github.com/kurator-dev/kurator/community/membership/community-membership.md
```
Comment (Xieql) [inline .github/ISSUE_TEMPLATE/membership-request.md:38]: link is not found

Comment (LiZhenCheng9527) [reply .github/ISSUE_TEMPLATE/membership-request.md:38]: wait #500 merge into main branch

Comment (Xieql) [reply .github/ISSUE_TEMPLATE/membership-request.md:39]: I see

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**